e2e/system_test.go

```diff
@@ -1801,43 +1802,59 @@ func TestChallenge(t *testing.T) {
 	securityCouncil, err := bindings.NewSecurityCouncilCaller(predeploys.DevSecurityCouncilAddr, l1Client)
 	require.NoError(t, err)
 
+	targetOutputOracleIndex := uint64(math.Ceil(float64(testdata.TargetBlockNumber) / float64(cfg.DeployConfig.L2OutputOracleSubmissionInterval)))
+
 	// set a timeout for one cycle of challenge
-	ctx, cancel := context.WithTimeout(context.Background(), 300*time.Second)
+	ctx, cancel := context.WithTimeout(context.Background(), 200*time.Second)
 	defer cancel()
 
 	ticker := time.NewTicker(1 * time.Second)
 	defer ticker.Stop()
 
-	targetOutputOracleIndex := uint64(math.Ceil(float64(testdata.TargetBlockNumber) / float64(cfg.DeployConfig.L2OutputOracleSubmissionInterval)))
-
-	for {
-		challengeStatus, err := colosseum.GetStatus(&bind.CallOpts{}, new(big.Int).SetUint64(targetOutputOracleIndex))
-		require.NoError(t, err)
-
-		// APPROVED status
-		if challengeStatus == chal.StatusApproved {
-			// check tx executed
-			tx, err := securityCouncil.Transactions(&bind.CallOpts{}, new(big.Int).SetUint64(0))
-			require.NoError(t, err)
-			require.Equal(t, tx.Executed, true)
-
-			// check challenge status is approved
-			challenge, err := colosseum.GetChallenge(&bind.CallOpts{}, new(big.Int).SetUint64(targetOutputOracleIndex))
-			require.NoError(t, err)
-			require.Equal(t, challenge.Approved, true)
-
-			// check output replaced by challenger
-			output, err := l2OutputOracle.GetL2Output(&bind.CallOpts{}, new(big.Int).SetUint64(targetOutputOracleIndex))
+	for ; ; <-ticker.C {
+		select {
+		case <-ctx.Done():
+			t.Fatalf("Timed out for challenge test")
+		default:
+			challengeStatus, err := colosseum.GetStatus(&bind.CallOpts{}, new(big.Int).SetUint64(targetOutputOracleIndex), cfg.Secrets.Addresses().Challenger1)
 			require.NoError(t, err)
-			require.Equal(t, output.Submitter, cfg.Secrets.Addresses().Challenger)
 
-			return
+			// wait until status becomes READY_TO_PROVE
+			if challengeStatus != chal.StatusReadyToProve {
+				continue
+			}
 		}
+		break
+	}
+	cancel()
+
+	// set a timeout for security council to validate output
+	ctx, cancel = context.WithTimeout(context.Background(), 10*time.Second)
```

Comment:
Also changed the time duration that security council validates output from 100 seconds to 10 seconds. I tested it is enough time for guardian to validate.